app/views/invitations/_invitation.html.erb

```diff
@@ -10,8 +10,9 @@
     <dd><%= invitation.invitee_email %></dd>
     <% else %>
     <dd>
+      <%= error_messages_for @invitation %>
```

Comment:
I poked around a bit to see where we usually put this, and it looks like most of the time it's either after the page heading (which we don't have here) or the first or second thing inside the form. Can we move this inside the `form_for`? 

You might also be able to use `invitation` rather than `@invitation` here.